packages/tailwind/modules/breakpoint.ts

```diff
@@ -0,0 +1,40 @@
+type VuesticThresholdsList = Record<string, number>
+
+/**
+ * @param thresholds tailwind css screens settings
+ * @returns vuestic ui thresholds settings
+ */
+export const convertThresholds = (thresholds: Record<string, string>): VuesticThresholdsList => {
+  return Object.fromEntries(
+    Object.entries(thresholds)
+      .map(([key, value]) => {
+        const vuesticValue = Number(value.substring(0, value.length - 2))
+        return [key, vuesticValue]
+      }),
+  )
+}
+
+/**
+ * @param tailwindConfig users or default tailwind config
+ * @returns vuestic breakpoint config
+ */
+export const proceedBreakpoint = (tailwindConfig?: Record<string, any>): VuesticThresholdsList => {
+  let tailwindThresholds
+
+  if (tailwindConfig) {
+    const tailwindThresholdsConfig = tailwindConfig.theme?.screens || tailwindConfig.theme?.extend?.screens
```

Comment:
Is it possible to have in tailwind config both of this?

Reply:
Yes, but this makes no sense - they're interchangeable (overwriting or extending default config -> if it's overwritten, we couldn't just extend).